x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality_panel/data_quality_details/indices_details/index.tsx

```diff
@@ -34,19 +35,62 @@ const IndicesDetailsComponent: React.FC<Props> = ({
   const { patternRollups, patternIndexNames } = useResultsRollupContext();
   const { patterns } = useDataQualityContext();
 
+  const [isTourActive, setIsTourActive] = useState<boolean>(() => {
+    const isActive = localStorage.getItem(HISTORICAL_RESULTS_TOUR_IS_ACTIVE_STORAGE_KEY);
+    return isActive !== 'false';
+  });
+
+  const handleDismissTour = useCallback(() => {
+    setIsTourActive(false);
+    localStorage.setItem(HISTORICAL_RESULTS_TOUR_IS_ACTIVE_STORAGE_KEY, 'false');
+  }, []);
```

Comment:
NIT: Sometimes it's easier to just use `useLocalStorage`. Also, it may be easier to store a _dismissed_ flag instead of _isActive_, this way, when there's no value in the storage it's more obvious to relate that it has not been dismissed.

```
const [isTourDismissed, setIsTourDismissed] = useLocalStorage<boolean>(HISTORICAL_RESULTS_TOUR_IS_DISMISSED_STORAGE_KEY, false);
```

Reply:
thanks will change

Reply:
done